src/xopen/__init__.py

```diff
@@ -373,7 +395,8 @@ def xopen(filename, mode='r', compresslevel=6, threads=None):
     will raise an error.
 
     compresslevel is the compression level for writing to gzip files.
-    This parameter is ignored for the other compression formats.
+    This parameter is ignored for the other compression formats. If set to
+    None (default) will choose the default compression level for the format.
```

Comment:
That the default compression level is 6 for gzip should be mentioned somewhere because of the difference to the `gzip.open` default.

```suggestion
    None (default), level 6 is used.
```